README.md

```diff
@@ -93,6 +94,7 @@ If you do not have a GPU, add the define "USE_CPU_ONLY", for example
 by adding -DUSE_CPU_ONLY=1 to the cmake command line.
 * Optional: BLAS Library: OpenBLAS (libopenblas-dev) or Intel MKL
 * The program has been tested on Windows, Linux and macOS.
+* autogtp and leelaz binaries need to be in the same directory. The below instructions can compile leelaz, and optionally autogtp at the same time.
```

Comment:
The instructions already do this, this serves no purpose.

Reply:
the purpose was in case end user somehow forgets to copy leelaz binary in same folder as autogtp binary

but it is right that if followed correctly, this information should not be needed, and may add confusion, will remove then